### PR TITLE
monitoring: add possible solution to echo_command_duration_test

### PR DESCRIPTION
**Pull request by unknwon**
Context: https://sourcegraph.slack.com/archives/C0J618TTM/p1603698207043400

### CODE REVIEW

File: doc/admin/observability/alert_solutions.md [hunk 1]
```diff
@@ -1005,6 +1005,7 @@ To learn more about Sourcegraph's alerting, see [our alerting documentation](htt
 
 **Possible solutions:**
 
+- **Query a graph for indivual commands** using "sum by (cmd)(src_gitserver_exec_running)" to gain more insights.
```
Comment (keegancsmith) [inline doc/admin/observability/alert_solutions.md:1008]:
```suggestion
- **Query a graph for individual commands** using "sum by (cmd)(src_gitserver_exec_running)" to gain more insights.
```

File: monitoring/git_server.go [hunk 2]
```diff
@@ -83,6 +83,7 @@ func GitServer() *Container {
 							PanelOptions:    PanelOptions().LegendFormat("running commands").Unit(Seconds),
 							Owner:           ObservableOwnerCloud,
 							PossibleSolutions: `
+								- **Query a graph for indivual commands** using "sum by (cmd)(src_gitserver_exec_running)" to gain more insights.
```
Comment (keegancsmith) [inline monitoring/git_server.go:86]:
```suggestion
								- **Query a graph for individual commands** using "sum by (cmd)(src_gitserver_exec_running)" to gain more insights.
```

**Review by keegancsmith — APPROVED** [review]

File: monitoring/git_server.go (hunk 2 above)
Comment (bobheadxi) [inline monitoring/git_server.go:86]:
```suggestion
								- **Query a graph for indivual commands** using 'sum by (cmd)(src_gitserver_exec_running)' in Grafana ('/-/debug/grafana') to see if a command might be spiking in frequency.
```

(single-quotes get replaced with markdown code backticks 🚀 )

Comment (unknwon) [reply monitoring/git_server.go:86]: >(single-quotes get replaced with markdown code backticks 🚀 )

TIL :)

File: doc/admin/observability/alert_solutions.md [hunk 3]
```diff
@@ -1005,6 +1005,7 @@ To learn more about Sourcegraph's alerting, see [our alerting documentation](htt
 
 **Possible solutions:**
 
+- **Query a graph for individual commands** using `sum by (cmd)(src_gitserver_exec_running)` to gain more insights.
```
Comment (unknwon) [inline doc/admin/observability/alert_solutions.md:1008]:
```suggestion
- **Query a graph for individual commands** using `sum by (cmd)(src_gitserver_exec_running)` in Grafana (`/-/debug/grafana`) to see if a command might be spiking in frequency.
```

File: monitoring/git_server.go [hunk 4]
```diff
@@ -83,6 +83,7 @@ func GitServer() *Container {
 							PanelOptions:    PanelOptions().LegendFormat("running commands").Unit(Seconds),
 							Owner:           ObservableOwnerCloud,
 							PossibleSolutions: `
+								- **Query a graph for individual commands** using `sum by (cmd)(src_gitserver_exec_running)` to gain more insights.
```
Comment (unknwon) [inline monitoring/git_server.go:86]:
```suggestion
								- **Query a graph for individual commands** using 'sum by (cmd)(src_gitserver_exec_running)' in Grafana ('/-/debug/grafana') to see if a command might be spiking in frequency.
```

**Pull request merged**